lib/controllers/pingController.js

```diff
@@ -1,4 +1,6 @@
-var logger = rootRequire('logging/logger');
+'use strict';
+
+var logger = require('lla-logger');  //!todo: change to use new logger
```

Comment:
If lla-logger is a singleton, than we can just require it with the () across files.